MatrixSDK/Crypto/Algorithms/Megolm/MXMegolmEncryption.m

```diff
@@ -198,7 +223,7 @@ - (MXHTTPOperation *)getDevicesInRoom:(NSArray<NSString*>*)users
                 }
 
                 if (deviceInfo.trustLevel.localVerificationStatus == MXDeviceBlocked
-                    || (!deviceInfo.trustLevel.isVerified && encryptToVerifiedDevicesOnly))
+                    || (!deviceInfo.trustLevel.isVerified && encryptToVerifiedDevicesOnly && !forceDistributeToUnverified))
```

Comment:
I think it would be better / cleaner to include this extra condition in the `BOOL encryptToVerifiedDevicesOnly = ` declaration.